gen/generator.jl

```diff
@@ -0,0 +1,46 @@
+using Pkg
+using Pkg.Artifacts
+using Clang.Generators
+using Clang.Generators.JLLEnvs
+using SuiteSparse_jll
+
+cd(@__DIR__)
```

Comment:
Is this really required? In that case perhaps it should be ina `cd(@__DIR__) do` block?

Reply:
I did this in case the working directory is not this folder. Those file paths in the toml config file are all relative to `@__DIR__`. This is useful when click `run the file` in vscode. What's the benefit of using `cd` with do-block?

Reply:
The only advantage really if you `include` this file (e.g. when debugging it) it doesn't change the working directory.